.circleci/config.yml

```diff
@@ -2,18 +2,18 @@ version: 2.1
 jobs:
   lint:
     docker:
-      - image: salsify/ruby_ci:2.6.10
+      - image: ruby:3.0.6
     working_directory: ~/goldiloader
     steps:
       - checkout
       - run:
```

Comment:
Can this install of SQLite be removed if we're pinning the gem to a corresponding older version?